docs/ansible_command.md

```diff
@@ -9,6 +9,8 @@ The `ansible` command is used to check ansible inventory for adherence to a sche
 
 If all checks pass, `schema-enforcer` will inform the user that all tests have passed.
 
+> NOTE | Schema enforcer does not come with ansible pre-installed, rather it is an optional dependency. The user can install schema enforcer bundled with ansible using one of `pip install schema-enforcer[ansible-base]` or `pip install schema-enforcer[ansible]`. Likewise, if ansible is already installed inside of the active python environment, the ansible package which is already installed will be used.
```

Comment:
Wonder if a similar note should be added to the main README? Thoughts?

Reply:
I think it's a good idea.